lib/plugins/aws/package/compile/events/sqs/index.js

```diff
@@ -12,9 +12,20 @@ class AwsCompileSQSEvents {
       'package:compileEvents': this.compileSQSEvents.bind(this),
     };
 
-    // TODO: Complete schema, see https://github.com/serverless/serverless/issues/8033
     this.serverless.configSchemaHandler.defineFunctionEvent('aws', 'sqs', {
-      anyOf: [{ type: 'string' }, { type: 'object' }],
+      oneOf: [
+        { $ref: '#/definitions/awsArn' },
+        {
+          type: 'object',
+          properties: {
+            arn: { $ref: '#/definitions/awsArn' },
+            batchSize: { type: 'integer', minimum: 1, maximum: 10000 },
```

Comment:
If I see correctly in AWS docs, for SQS maximum is `10` -> https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-lambda-eventsourcemapping.html#cfn-lambda-eventsourcemapping-batchsize